docs/user/tutorial/60-admin.rst

```diff
@@ -81,10 +113,24 @@ Editing items
 You can edit a file's details by selecting it.
 
 * select one of the existing images
+
+.. image:: images/open_image.png
+   :alt: click on the image
+   :width: 80%
+
 * edit ``Name`` and ``Description``
+
+.. image:: images/image_description.png
+   :alt: Shows image information
+   :width: 80%
+
 * Choose the image's *focus point* - the point around which any cropping will occur - by dragging
   the red circle in the image pane
 
+.. image:: images/image_focus.png
+   :alt: red dot on image
+   :width: 80%
```

Comment:
The ``image`` directives like this one that you have added document are part of the list items, therefore they should be indented by the same amount as the list content.

In the case of this one, the indentation of the following paragraph means it raises an error. It should look like:

```
*   Choose the image's *focus point* - the point around which any cropping will occur - by dragging
    the red circle in the image pane

    .. image:: images/image_focus.png
       :alt: red dot on image
       :width: 80%

    This helps ensure for example that however a portrait is cropped, the subject's head will not be
    chopped off. If your image doesn't have a particular focus point, leave the circle in the middle
    of the image.
```